app/configurator/config-form.tsx

```diff
@@ -487,19 +469,14 @@ export const useMultiFilterCheckboxes = (
     [dispatch, dimensionIri, allValues, value, onChangeProp]
   );
 
-  const checkedState =
+  const isChecked =
     state.state === "CONFIGURING_CHART"
       ? isMultiFilterFieldChecked(state.chartConfig, dimensionIri, value)
       : false;
```

Comment:
nit: I think this would be more clear:

```ts
const isChecked = state.state === "CONFIGURING_CHART" && isMultiFilterFieldChecked(state.chartConfig, dimensionIri, value);
```